vignettes/admiral.Rmd

```diff
@@ -38,10 +42,26 @@ ds <- admiral_ds
 ex <- admiral_ex
 ```
 
-```{r, eval=TRUE}
-# derive treatment variables
+## Derive Treatment Variables (`TRT0xP`, `TRT0xA`) {#treatmentvargs}
+
+The mapping of the treatment variables is left to the ADaM programmer. An example mapping may be:
+
+```{r eval=TRUE}
 adsl <- dm %>%
-  mutate(TRT01P = ARM, TRT01A = ACTARM) %>%
+  mutate(TRT01P = ARM, TRT01A = ACTARM)
+```
+
+## Derive/Impute Numeric Treatment Date/Time and Duration (`TRTSDTM`, `TRTEDTM`) {#trtdatetimegs}
+
+The function `derive_vars_merged_dtm()` can be used to derive the treatment
+start and end date/times using the `EX` domain. This call returns the original data frame with the column `TRTSDTM` and`TRTEDTM` added. Exposure observations with incomplete date and zero doses of non placebo treatments are ignored. Missing time parts are imputed as first or
```

Comment:
Replace `This call returns` with `The function calls below` (don't mind if you leave as is, just a suggestion)

Reply:
updated